_gsocproposals/proposal_GoHEPInterpreter.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Launching Wagon, a WebAssembly interpreter in Go
+layout: plain
+project: Go-HEP
+organization: CERN
```

Comment:
@sbinet is the organization really CERN? You may want to create another one after #88 has been merged.